### PR TITLE
Kruskal algorithm in java

### PR DESCRIPTION
**Pull request by shraiyya**
The code of kruskal algorithm is written in java.
The sample output is pasted in the comments.

### CODE REVIEW

**Comment by CaioCordeiro** [comment]
What a great code :smile: !  
Thank you for contributing! :rocket:

File: Kruskal Algorithm/kruskal.java [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+5| 4 0 0 9 0 0  
+6| 0 7 8 0 0 0  
+ 
+EDGE 0:(1, 4) COST:1 
+EDGE 1:(1, 2) COST:2 
+EDGE 2:(2, 3) COST:3 
+EDGE 3:(1, 5) COST:4 
+EDGE 4:(2, 6) COST:7 
+MINIMUM COST=  17
+*/
```
Comment (CaioCordeiro) [inline Kruskal Algorithm/kruskal.java:233]: Maybe its a good idea to just put a input sample in this comment :grin:

**Review by CaioCordeiro — CHANGES_REQUESTED** [review]

File: Kruskal Algorithm/kruskal.java (hunk 1 above)
Comment (shraiyya) [reply Kruskal Algorithm/kruskal.java]: Yes sure.
I will do it!

**Comment by shraiyya** [comment]
I have now only put an input sample in this comment.

**Comment by CaioCordeiro** [comment]
Awesome :smile: ! Thats a great Patch!
Thank you for contributing! :rocket:

**Pull request merged**